pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -179,6 +191,16 @@ func StatefulSetForCluster(
 									Name:  "CASSANDRA_RACK",
 									Value: "Rack1-K8Demo",
 								},
+								{
+									Name: "JVM_OPTS",
+									Value: fmt.Sprintf(
+										"-javaagent:%s/jolokia.jar=host=%s,port=%d,agentContext=%s",
+										sharedVolumeMountPath,
+										jolokiaHost,
+										jolokiaPort,
+										jolokiaContext,
+									),
+								},
```

Comment:
This is okay for now, but I think this environment variable should be set/managed by Pilot in future. We've got a similar question/issue coming up in the ES controller, as we need to see the heap size (via env var) too.

Reply:
(line comment went weird - this is specifically around setting the JAVA_OPTS env var)